meta-frc971/recipes-frc971/configuration/frc971-dotfiles/contents/etc/fstab

```diff
@@ -0,0 +1,8 @@
+# /etc/fstab: static file system information.
+#
+# Use 'blkid' to print the universally unique identifier for a
+# device; this may be used with UUID= as a more robust way to name devices
+# that works even if disks are added and removed. See fstab(5).
+#
+# <file system> <mount point>   <type>  <options>       <dump>  <pass>
+/dev/nvme0n1p1 / xfs rw,noatime,discard 0 0
```

Comment:
Can you help teach me how the root filesystem ends up as XFS?  My recollection is yocto uses ext4 by default.

We probably have options if it gets hard, though I've done enough benchmarking that I'm nervous about ext4 when writing large logs.

Reply:
I found a way to fix this by making a script which converts the fs, will get this checked in.

Reply:
Currently have issues getting xfs to work. Have been spending the last few weeks on it. I haven't been able to switch the filesystem over to xfs successfully but we have other higher priority issues to work on and we're getting fine image logs out of ext4 so I will come back to this when some of the higher priority items are done.